page-objects/copyCourse.js

```diff
@@ -65,10 +65,9 @@ module.exports = {
 	addText: async function(text) {
 		let textBtn = ".btn-group > button:nth-child(1)";
 		await helpers.waitAndClick(textBtn);
-		let iframe = await driver.$('#cke_1_contents > iframe');
 		let textField = await driver.$('body');
-		await iframe.click();
-		await driver.switchToFrame(iframe);
+		await driver.pause(2000);
```

Comment:
I assume the predefined pause is an experimental value based on environment. Please extract it as global configuration value either in a new ticket or now. You may use our commons.configuration for it.

Reply:
I would like to that changes in another branch - as a colleague needs my changes.